src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -256,7 +274,8 @@ public static HttpHeaders generateResponseEntityWithPaginationHeader(Pageable pa
         if (responseEntity == null) {
             headers.add("paginationInfo", pageInformation.toString());
         } else {
-            // since Headers are read only , it can't be modified and hence copied all existing headers into new one and added new header for pagination
+            // since Headers are read only , it can't be modified and hence copied all existing headers into new
+            // one and added new header for pagination

```

Comment:
Consider changing this comment to:

            // Since Headers are read only, they can't be modified. 
            // Hence, copying all existing headers into new one and added new header for pagination